### DISCUSSION TITLE
Reading WSI Pathology slides

### DISCUSSION THREAD

**Discussion by vikashg** [Project-MONAI/MONAI#3737] [Q&A]
Hello everyone, I am looking for a python based reader where I can read whole slide images from pathology. I looked at several programs like OpenSlide but I always run into errors. Anyone knows about a good io for WSIs?

Thanks 
Vikash

**Comment by Nic-Ma** [comment]
Hi @vikashg ,

I think you can refer to:
https://github.com/Project-MONAI/tutorials/tree/master/pathology
There is some usage example of MONAI WSIReader.

Thanks.
